taskqueue/gcloud/aio/taskqueue/__init__.py

```diff
@@ -2,12 +2,12 @@
 __version__ = get_distribution('gcloud-aio-taskqueue').version
 
 from gcloud.aio.taskqueue.pullqueue import PullQueue
+from gcloud.aio.taskqueue.pushqueue import PushQueue
 from gcloud.aio.taskqueue.taskmanager import FailFastError
 from gcloud.aio.taskqueue.taskmanager import TaskManager
-from gcloud.aio.taskqueue.taskqueue import TaskQueue
```

Comment:
As discussed on dialpad, please leave this in. We'll remove deprecations separately :)

Reply:
oh sorry, didn't realize I deleted it!